src/test/java/mate/academy/springbootintro/service/BookServiceTest.java

```diff
@@ -115,22 +127,22 @@ public void getAll_ValidPageable_ReturnsAllBooks() {
         // Given
         Category category1 = new Category();
         category1.setId(CATEGORY_ID_1);
-        category1.setDescription("fantasy");
-        category1.setDescription("dragons");
+        category1.setName(CATEGORY_NAME_1);
+        category1.setDescription(CATEGORY_DESCRIPTION_1);
 
         Category category2 = new Category();
         category2.setId(CATEGORY_ID_2);
-        category2.setName("action");
-        category1.setDescription("pif-paf");
+        category2.setName(CATEGORY_NAME_2);
+        category1.setDescription(CATEGORY_DESCRIPTION_2);
 
         Book book = new Book();
         book.setId(BOOK_ID);
-        book.setTitle("Godzilla");
-        book.setAuthor("Vermud");
-        book.setIsbn("123456789");
-        book.setPrice(BigDecimal.TEN);
-        book.setCoverImage("random_cover2.jpg");
-        book.setDescription("big monster");
+        book.setTitle(BOOK_TITLE);
+        book.setAuthor(BOOK_AUTHOR);
+        book.setIsbn(BOOK_ISBN);
+        book.setPrice(BOOK_PRICE);
+        book.setCoverImage(BOOK_COVER_IMAGE);
+        book.setDescription(BOOK_DESCRIPTION);

```

Comment:
You should create and use something similar to `createBookDto` you created in BookControllerTest